mixer/test/client/env/envoy.go

```diff
@@ -28,8 +28,8 @@ import (
 )
 
 const (
-	liveTimeout = 10 * time.Second
-	waitTimeout = 3 * time.Second
+	liveTimeout = 30 * time.Second
+	waitTimeout = 15 * time.Second
```

Comment:
Envoy initialization is very likely  to be timeout in local dev environment with old timeout. I increased them a bit.